docs/download/guide.md

```diff
@@ -538,7 +538,7 @@ Related terms and their definitions are given below:
 | _Compute Instance_  | A user virtual machine operating in a cloud environment.                                                                                                                                                                                                                                 |
 | _DACO_              | The Data Access Compliance Office which handles requests from researchers for access to controlled data from the ICGC.                                                                                                                                                                   |
```

Comment:
@hknahal can you please change to `ICGC DACO` and `The ICGC Data Access Compliance Office`
and link them to
to https://daco.icgc-argo.org/